common/src/main/java/com/microsoft/identity/common/internal/logging/CommonCoreLogger.java

```diff
@@ -0,0 +1,264 @@
+package com.microsoft.identity.common.internal.logging;
+
+import android.os.Build;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import com.microsoft.identity.common.adal.error.ADALError;
+import com.microsoft.identity.common.adal.internal.util.StringExtensions;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+import java.util.concurrent.atomic.AtomicReference;
+
+public final class CommonCoreLogger {

```

Comment:
Let's also plan to avoid prefixing this class with `CommonCore` -- let's just call it `Logger`

Reply:
Categorically, we shouldn't let class names in our dependencies prohibit us from choosing "good" (quotes, because subjective) class names in our library. If you're of a mind that `CommonCoreLogger` is a better than name than `Logger` that's something we can work with. If you'd like to call it `Logger`, consider using the fully qualified name to dereference it: `com.microsoft.identity.common.internal.logging.Logger` (or whichever class name)